apps/src/weblab/main.js

```diff
@@ -0,0 +1,19 @@
+var appMain = require('../appMain');
+var studioApp = require('../StudioApp').singleton;
+var WebLab = require('./WebLab');
+// var blocks = require('./blocks');
+var skins = require('../skins');
+// var levels = require('./levels');
```

Comment:
Good points both. Have done both. It does make Web Lab's main.js different than other apps', but we have already deliberately diverged in this PR and have ES6-ized the rest of Web Lab (in code that is very similar to other apps). So might as well go all-in on ES6 in Web Lab!